sw/device/lib/base/status.h

```diff
@@ -56,48 +56,45 @@ typedef struct status {
   int32_t value;
 } status_t;
 
+#define DIF_RESULT_INTO_STATUS(_exp)                                      \
```

Comment:
I forgot  to comment on this:  names prefixed with `_` are reserved by the C standard.  The original version of the INTO_STATUS macro used names suffixed with an underscore.